### PR TITLE
Feature/oped id connect removal

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: api/auth/index.ts [hunk 1]
```diff
@@ -1,195 +1,179 @@
...
-router.get('/logout', async (req: Request, res: Response) => {
-    await doLogout(req, res)
-})
+export async function oauth(req: EnhancedRequest, res: express.Response, next: express.NextFunction) {
+    const response = await getTokenFromCode(req, res)
+    const accessToken = response.data.access_token
+
+    if (accessToken) {
+        const userDetails = await asyncReturnOrError(getUserDetails(accessToken, idamUrl), 'Cannot get user details', res, logger, false)
+        const isPrdAdminRole = havePrdAdminRole(userDetails)
```
Comment (paddybasi) [inline api/auth/index.ts]: When reintegrating oidc, remember this code as it was lost in transport

File: config/custom-environment-variables.json [hunk 2]
```diff
@@ -13,10 +12,7 @@
   "maxLogLine": "MAX_LOG_LINE",
   "idamClient": "IDAM_CLIENT",
   "microservice": "MICROSERVICE",
-  "now": {
-    "__name": "NOW",
-    "__format": "json"
-  },
+  "now": "NOW",
```
Comment (paddybasi) [inline config/custom-environment-variables.json]: careful here, still need this fix in place

File: package.json [hunk 3]
```diff
@@ -32,8 +32,8 @@
     "test": "yarn test:ng",
     "test:smoke": "sh whitelist.sh",
     "test:smoke:local": "webdriver-manager update --versions.chrome 2.40 && protractor ./test/e2e/config/smoke.conf.js --local",
-    "test:functional:backup": "echo 'Not implemented'",
-    "test:functional": "webdriver-manager update --versions.chrome 2.40 && protractor ./test/e2e/config/conf.js",
+    "test:functional": "echo 'Not implemented'",
+    "test:functional:backup": "webdriver-manager update --versions.chrome 2.40 && protractor ./test/e2e/config/conf.js",
```
Comment (paddybasi) [inline package.json]: undo these changes as it's effectively disabling functional tests?

File: src/app/components/terms-and-conditions/terms-and-conditions.component.html [hunk 4]
```diff
@@ -3,40 +3,84 @@
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-xl">Terms and conditions</h1>
-        <p class="govuk-body">By accessing this service (‘the Service’), you’re agreeing to the following terms of use. This includes the <a routerLink="/privacy-policy">privacy policy</a>.</p>
-        <p class="govuk-body">The use of the Service includes the Manage Organisation and Manage Cases applications (sometimes known as ‘MyHMCTS’) and Judicial Case Manager (JCM).</p>
-        <p class="govuk-body">The term ‘you’ includes yourself and all users of the Service in your organisation.  The terms ‘we’ and ‘us’ refer to Her Majesty’s Courts and Tribunals Service.</p>
-        <p class="govuk-body">You also agree that you have a legitimate need to use the Service.</p>
+        <p class="govuk-body">By using this service you're agreeing to these terms of use. This includes the <a routerLink="/privacy-policy">privacy policy</a>.</p>
```
Comment (paddybasi) [inline src/app/components/terms-and-conditions/terms-and-conditions.component.html]: check which is latest T&C's and update accordingly

File: src/app/utils/app-utils.ts [hunk 5]
```diff
@@ -94,31 +95,4 @@ export class AppUtils {
 
     return organisations;
   }
-
-  public static capitalizeString(stringToCapitalize: string) {
```
Comment (paddybasi) [inline src/app/utils/app-utils.ts]: not sure if/when this block of code went in, but ensure it isn't needed

Comment (udaydenduluri33) [reply src/app/utils/app-utils.ts]: @paddybasi  just verified both functions are not being used anywhere. Thanks.

File: src/app/components/terms-and-conditions/terms-and-conditions.component.html (hunk 4 above)
Comment (udaydenduluri33) [reply src/app/components/terms-and-conditions/terms-and-conditions.component.html]: Done @paddybasi

File: package.json (hunk 3 above)
Comment (udaydenduluri33) [reply package.json]: Done @paddybasi

File: config/custom-environment-variables.json (hunk 2 above)
Comment (udaydenduluri33) [reply config/custom-environment-variables.json]: Done @paddybasi
